runtime/src/comm/ugni/comm-ugni.c

```diff
@@ -890,6 +891,10 @@ typedef struct {
   void*            res;                 // if non-NULL, where to send result
 } fork_amo_info_t;
 
+typedef struct {
```

Comment:
You could actually get rid of this and just use a `fork_base_info_t` for shutdown AMs but if you want to keep it that's fine too, and indeed there may be a need for it when orderly shutdowns can be initiated by any node.

Reply:
Ah interesting and good to know. I'll probably leave it, it doesn't add much extra code and like you said we may want it in the future